sdk/JavaScript/TestFramework/js/testFramework.js

```diff
@@ -36,15 +36,15 @@ function createZumoNamespace() {
         this.canRunUnattended = true;
         this.logs = [];
         this.requiredFeatures = [];
-        if (requiredRuntimeFeatures) {
-            if (typeof requiredRuntimeFeatures === 'string') {
-                requiredRuntimeFeatures = [requiredRuntimeFeatures];
+        if (requiredRuntimeFeatures) {
+            if (typeof requiredRuntimeFeatures === 'string') {
+                requiredRuntimeFeatures = [requiredRuntimeFeatures];
             }
             var that = this;
-            requiredRuntimeFeatures.forEach(function (feature) {
-                that.requiredFeatures.push(feature);
-            });
-        }
+            requiredRuntimeFeatures.forEach(function (feature) {
+                that.requiredFeatures.push(feature);
+            });
```

Comment:
Very true, however this is existing code that I did not touch (beyond the whitespace) & I think that it is readable as is.

I'll keep this in mind for the future